### PR TITLE
Drop notebook functionality from upstream k8s version

### PR DESCRIPTION
**Pull request by michalkulakowski**

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -89,16 +96,22 @@ CATALOG_REPOSITORY ?= registry.toolbox.iotg.sclab.intel.com/cpp/openvino-operato
...
+	sed -i "s|registry.connect.redhat.com/intel/ovms-operator:1.0.0|$(OPERATOR_IMAGE):$(IMAGE_TAG)|" bundle/manifests/openvino-operator.clusterserviceversion.yaml
+	sed -i "s|gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0|registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.9.0|" bundle/manifests/openvino-operator.clusterserviceversion.yaml
+	docker build -t $(BUNDLE_REPOSITORY):$(IMAGE_TAG) -f bundle/Dockerfile  bundle
 	sed -i "s|$(OPERATOR_IMAGE):$(IMAGE_TAG)|registry.connect.redhat.com/intel/ovms-operator:0.2.0|" bundle/manifests/openvino-operator.clusterserviceversion.yaml
-	sed -i "s|registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.8.0|gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0|" bundle/manifests/openvino-operator.clusterserviceversion.yaml
+	sed -i "s|registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.9.0|gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0|" bundle/manifests/openvino-operator.clusterserviceversion.yaml
 else
 	echo "Building kubernetes bundle"
-	sed -i "s|registry.connect.redhat.com/intel/ovms-operator:0.2.0|$(OPERATOR_IMAGE):$(IMAGE_TAG)|" bundle/manifests/openvino-operator.clusterserviceversion.yaml
+ifeq ($(ADD_NOTEBOOK_K8S), 0)
```
Comment (mzegla) [inline Makefile:106]: I know it's a minor, but can we switch this?
So it's ADD_NOTEBOOK_K8S != 0 or something. 

I feel like IF <NON_DEFAULT>: <some specific stuff>; ELSE <default path>
is a bit more readible.

File: bundle/Dockerfile [hunk 2]
```diff
@@ -33,6 +33,7 @@ LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
 LABEL com.redhat.openshift.versions="v4.6"
 
+COPY manifests/intel.com_model_servers.yaml manifests/openvino-operator-controller-manager_v1_serviceaccount.yaml manifests/openvino-operator-controller-manager-metrics-service_v1_service.yaml manifests/openvino-operator-manager-config_v1_configmap.yaml manifests/openvino-operator-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml /manifests/
```
Comment (mzegla) [inline bundle/Dockerfile:36]: Is it required? Are we not copying all `manifests` in the line below?

File: docs/developer_guide.md [hunk 3]
```diff
@@ -22,6 +22,11 @@ Start the operator
 make run
 ```
 
+Start the operator
```
Comment (mzegla) [inline docs/developer_guide.md:25]:
```suggestion
Start the operator for Kubernetes
```

File: docs/developer_guide.md [hunk 4]
```diff
@@ -41,17 +46,17 @@ make deploy IMG=registry.toolbox.iotg.sclab.intel.com/cpp/operator:latest
 ## OLM development flow
 ```
 make cluster_clean
-make build_all_images
+make build_all_images DROP_NOTEBOOK=0
```
Comment (mzegla) [inline docs/developer_guide.md:49]: I think it's ADD_NOTEBOOK_K8S now and it's 0 by default so maybe we can drop this?

**Review by mzegla — APPROVED** [review]

**Pull request merged**